src/Reservations/docs/Get-AzReservationOrderId.md

```diff
@@ -0,0 +1,125 @@
+---
+external help file:
+Module Name: Az.Reservations
+online version: https://docs.microsoft.com/powershell/module/az.reservations/get-azreservationorderid
+schema: 2.0.0
+---
+
+# Get-AzReservationOrderId
+
+## SYNOPSIS
+Get applicable `Reservation`s that are applied to this subscription or a resource group under this subscription.
+
+## SYNTAX
+
+### Get (Default)
+```
+Get-AzReservationOrderId [-SubscriptionId <String[]>] [-DefaultProfile <PSObject>] [<CommonParameters>]
+```
+
+### GetViaIdentity
+```
+Get-AzReservationOrderId -InputObject <IReservationsIdentity> [-DefaultProfile <PSObject>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Get applicable `Reservation`s that are applied to this subscription or a resource group under this subscription.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```

Comment:
Could you please check your examples and make sure all placeholders are filled?

Reply:
Filled the missing titles